src/platform_impl/ios/view.rs

```diff
@@ -213,30 +240,36 @@ unsafe fn get_window_class() -> &'static Class {
                 let uiscreen = msg_send![object, screen];
                 let touches_enum: id = msg_send![touches, objectEnumerator];
                 let mut touch_events = Vec::new();
+                let supports_force = app_state::capabilities().force_touch;
                 loop {
                     let touch: id = msg_send![touches_enum, nextObject];
                     if touch == nil {
                         break;
                     }
                     let location: CGPoint = msg_send![touch, locationInView: nil];
                     let touch_type: UITouchType = msg_send![touch, type];
-                    let trait_collection: id = msg_send![object, traitCollection];
-                    let touch_capability: UIForceTouchCapability =
-                        msg_send![trait_collection, forceTouchCapability];
-                    let force = if touch_capability == UIForceTouchCapability::Available {
-                        let force: CGFloat = msg_send![touch, force];
-                        let max_possible_force: CGFloat = msg_send![touch, maximumPossibleForce];
-                        let altitude_angle: Option<f64> = if touch_type == UITouchType::Pencil {
-                            let angle: CGFloat = msg_send![touch, altitudeAngle];
-                            Some(angle as _)
+                    let force = if supports_force {
+                        let trait_collection: id = msg_send![object, traitCollection];
+                        let touch_capability: UIForceTouchCapability =
```

Comment:
It might be good to mention that this is checking for _device-level_ support, and to make it more explicit that `Capabilities` is specifically OS-level support (which would probably also involve adding some comments on `Capabilities`).